cpp/tests/io/parquet_writer_test.cpp

```diff
@@ -1407,17 +1383,62 @@ TEST_P(CompressionTest, CompStatsEmptyTable)
   expect_compression_stats_empty(stats);
 }
 
-INSTANTIATE_TEST_CASE_P(NvcompCompressionTest,
-                        CompressionTest,
+TEST_P(ParquetCompressionTest, RoundTripBasic)
+{
+  constexpr auto num_rows     = 12000;
+  auto const compression_type = std::get<1>(GetParam());
+
+  // Generate compressible data
+  auto int_sequence =
+    cudf::detail::make_counting_transform_iterator(0, [](auto i) { return i % 100; });
+  auto float_sequence =
+    cudf::detail::make_counting_transform_iterator(0, [](auto i) { return i / 32; });
```

Comment:
using this instead of create_compressible_fixed_table to ensure dict encoding is used